site/content/blog/2020-12-01-whats-new-in-svelte-december-2020.md

```diff
@@ -0,0 +1,65 @@
+---
+title: What's new in Svelte: December 2020
+description: TBD
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+It's the last "What's new in Svelte" of the year... To round out the season, this month's coverage includes _things_!
+
+## New features & impactful bug fixes
+
+1. Transitions within `{:else}` block should now complete successfully (**3.29.5**, [Example](https://svelte.dev/repl/49cef205e5da459594ef2eafcbd41593?version=3.29.5))
+2. Svelte now includes an export map, which explicitly states which files can be imported from its npm package (**3.29.5** with some fixes in **3.29.6** and **3.29.7**)
+
```

Comment:
`rollup-plugin-svelte` had a [new 7.0.0 release](https://github.com/sveltejs/rollup-plugin-svelte/blob/master/CHANGELOG.md). Biggest change is that the `css` option was removed. Users who were using that option should add another plugin like `rollup-plugin-css-only` [as demonstrated in the template](https://github.com/sveltejs/template/blob/5b1135c286f7a649daa99825a077586655051649/rollup.config.js#L48)